entity-view/impl/src/main/java/com/blazebit/persistence/view/impl/mapper/ViewMapper.java

```diff
@@ -113,7 +114,8 @@ public ViewMapper(ManagedViewType<S> sourceType, ManagedViewType<T> targetType,
                 // Extract a mapping from target dirty state index to the source
                 int dirtyStateIndex = ((AbstractMethodAttribute<?, ?>) targetAttribute).getDirtyStateIndex();
                 if (dirtyStateIndex != -1) {
-                    MethodAttribute<? super S, ?> sourceAttribute = sourceType.getAttribute(targetAttribute.getName());
+                    // Does dirty state need to be copied from entities? Can this be done without modifying the entity class?
+                    MethodAttribute<? super S, ?> sourceAttribute = (sourceType == null) ? null :sourceType.getAttribute(targetAttribute.getName());
```

Comment:
I'm assuming copying dirty state from an entity would be out of scope

Reply:
Right, that's simply not possible.